### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by foekinator656**

### CODE REVIEW

**Comment by DragonTamer117** [comment]
Please wait till tomorrow for me to review this, I have to update my project.
Right now I can't review, since I'm having some internet issues and I am answering this through my phone.

File: src/components/admin/login/register/register.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import axios from "axios";
+import router from "@/router";
+
+export default {
+    name: "register-component",
+    data() {
+        return {
+            email: "",
+            password: "",
+            role: 0,
```
Comment (DragonTamer117) [inline src/components/admin/login/register/register.vue]: Role is a string that the backend pushes out from.
Please change this to:
`role: ""`

File: src/components/shared/ApiService.ts
Comment (DragonTamer117) [inline src/components/shared/ApiService.ts:1]: This file seems redundant, please remove it.

File: src/components/shared/order/order-item/order-item.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<template>
+<!--  <a-->
```
Comment (DragonTamer117) [inline src/components/shared/order/order-item/order-item.vue]: The HTML here is commented, please remove this.

File: src/components/shared/order/order.vue
Comment (DragonTamer117) [inline src/components/shared/order/order.vue:1]: Remove commented code here as well.

File: src/components/shared/order/payment/payment.vue
Comment (DragonTamer117) [inline src/components/shared/order/payment/payment.vue:1]: Remove comments.

File: src/router.js
Comment (DragonTamer117) [inline src/router.js:1]: Remove comments.

**Review by DragonTamer117 — CHANGES_REQUESTED** [review]

File: src/router.js
Comment (foekinator656) [reply src/router.js:1]: i am keeping the comments for later

**Comment by foekinator656** [comment]
@DragonTamer117 check again please

File: src/router.js
Comment (DragonTamer117) [reply src/router.js:1]: That's okay

**Review by DragonTamer117 — COMMENTED** [review]
LGTM!

**Review by DragonTamer117 — APPROVED** [review]
LGTM now, wrong branch. So some changes were not needed, next time watch the pull request to what branch.

**Pull request merged**